test/javascripts/response_test.coffee

```diff
@@ -55,3 +65,28 @@ describe 'TurboGraft.Response', ->
       responseForFixture 'serverError', (response) ->
         assert.equal(response.document(), undefined)
         done()
+
+  describe 'redirectedTo', ->
+    it 'returns the value of the X-XHR-Redirected-To header', (done) ->
+      responseForFixture 'xhrRedirectedToHeader', (response) ->
+        assert.equal(response.redirectedTo(), ROUTES['xhrRedirectedToHeader'][1]['X-XHR-Redirected-To'])
+        done()
+
+  describe 'redirectedToNewUrl', ->
+    beforeEach ->
+      sandbox.stub(TurboGraft.Response, 'currentLocation', -> 'test-location')
```

Comment:
I guess you can't stub the native window location, but this doesn't feel like the right place to have a method for accessing this. I would say either 1. Force the consuming method to accept a location, so that you can just pass it in instead of reading the global, or 2) Add another object that has a method for returning the current location that is more appropriate than the response.